docs/pages/versions/unversioned/sdk/splash-screen.mdx

```diff
@@ -97,7 +177,16 @@ See how to configure the native projects in the [installation instructions in th
 
 ### Animating the splash screen
 
-See the [with-splash-screen](https://github.com/expo/examples/tree/master/with-splash-screen) example on how to apply any arbitrary animations to your splash screen, such as a fade out. You can initialize a new project from this example by running `npx create-react-native-app -t with-splash-screen`.
+Expo splashscreen provides an out of the box fade animation. It can be configured using the `setOptions` method.
```

Comment:
```suggestion
`SplashScreen` provides an out-of-the-box fade animation. It can be configured using the `setOptions` method.
```

Let's use the module name here.